src/_locales/fr/messages.json

```diff
@@ -89,5 +89,9 @@
 
   "optionsDarkTheme": {
     "message": "Thème sombre"
+  },
+
+  "useRegExpFilter": {
+    "message": "Allow use regular expression to filter"
```

Comment:
Only add the string in the English file (well, and in your case the zh_CN one), if the string doesn't exist the localization framework will automatically fall back to the closest language.